lib/livebook_web/live/app_session_live.ex

```diff
@@ -94,6 +94,46 @@ defmodule LivebookWeb.AppSessionLive do
     """
   end
 
+  def render(%{data_view: %{output_type: :output_panel}} = assigns)
+      when assigns.app_authenticated? do
+    ~H"""
+    <div class="h-full w-full" data-el-output-panel>
+      <div data-el-js-view-iframes phx-update="ignore" id="js-view-iframes"></div>
+      <div class="flex items-center pb-4 mb-2 space-x-4 border-b border-gray-200 pr-20 md:pr-0">
+        <h1 class="text-3xl font-semibold text-gray-800">
+          <%= @data_view.notebook_name %>
+        </h1>
+      </div>
+      <div id="output-panel" class="flex flex-col" data-el-output-panel-content>
+        <%= for {output_row, row_index} <- Enum.with_index(@data_view.output_panel_views.rows) do %>
+          <div class="flex flex-grow" data-row-index={row_index} data-el-output-panel-row>
+            <div
+              :for={{item, col_index} <- Enum.with_index(output_row.items)}
+              id={"output-panel-item-#{row_index}-#{col_index}"}
+              class="relative group"
+              style={"width: #{item.width}%"}
```

Comment:
class={"w-[#{item.width}%]"} is not working. Does it has something to do with the tailwind JIT compiler?

Reply:
Most likely. You can try adding this to the template `<%!-- w-25% w-12.5% w-33.3% --%>` and it may pick it up.

Reply:
I see, but then I think using style is the better solution :)